### ISSUE TITLE
Bug after update. April 25, 2017

### ISSUE THREAD

**Issue by RealidadeMaromba** [woocommerce/woocommerce#14666]
### EXPLANATION OF THE ISSUE

After the update the title of the advertisement got big and in the each product the box resized. 
I tried resize the font size in the settings but it did not work. 

### STEPS TO REPRODUCE THE ISSUE

http://imgur.com/a/CqDQZ
http://loja.realidademaromba.com.br/
### SYSTEM STATUS

<details>
```
`
### WordPress Environment ###

Home URL: http://loja.realidademaromba.com.br
Site URL: http://loja.realidademaromba.com.br
WC Version: 3.0.4
Log Directory Writable: ✔
WP Version: 4.7.4
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: –
WP Cron: ✔
Language: pt_BR

### Server Environment ###

Server Info: Apache
PHP Version: ❌ 5.3.29 - Recomendamos PHP 5.6 como versão mínima. Veja: Como atualizar a versão do seu PHP
PHP Post Max Size: 8 MB
PHP Time Limit: 30
PHP Max Input Vars: 1000
cURL Version: 7.38.0
OpenSSL/1.0.0

SUHOSIN Installed: –
MySQL Version: ❌ 5.5.54 - Nós recomendamos utilizar pelo menos a versão 5.6 do MySQL. Mais informações: Requerimentos do WordPress
Max Upload Size: 2 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ❌ wp_remote_post() falhou. Entre em contato com a sua hospedagem. cURL error 35: Unsupported SSL protocol version
Remote Get: ❌ wp_remote_get() falhou. Entre em contato com a sua hospedagem. 200

### Database ###

WC Database Version: 3.0.4
WC Database Prefix: wp_
woocommerce_sessions: ✔
woocommerce_api_keys: ✔
woocommerce_attribute_taxonomies: ✔
woocommerce_downloadable_product_permissions: ✔
woocommerce_order_items: ✔
woocommerce_order_itemmeta: ✔
woocommerce_tax_rates: ✔
woocommerce_tax_rate_locations: ✔
woocommerce_shipping_zones: ✔
woocommerce_shipping_zone_locations: ✔
woocommerce_shipping_zone_methods: ✔
woocommerce_payment_tokens: ✔
woocommerce_payment_tokenmeta: ✔
MaxMind GeoIP Database: ❌ O banco de dados GeoIP MaxMind não existe. A geolocalização não funcionará. Você pode fazer download e o instalar manualmente de http://dev.maxmind.com/geoip/legacy/geolite/ para o caminho: . Role a página para baixo para "Downloads" e faça download do arquivo "Binary / gzip" ao lado de "GeoLite Country". Lembre-se de descompactar o GeoIP.dat.gz e fazer upload apenas do arquivo GeoIP.dat.

### Security ###

Secure connection (HTTPS): ❌Sua loja não está usando HTTPS. Saiba mais sobre HTTPS e certificados SSL.
Hide errors from visitors: ✔

### Active Plugins (4) ###

WooCommerce Correios: por Claudio Sanches – 3.2.2
WooCommerce PagSeguro: por Claudio Sanches – 2.12.4
WooCommerce: por Automattic – 3.0.4
WooCommerce Helper: por WooCommerce – 1.7.2 – Rede habilitada

### Settings ###

API Enabled: ✔
Force SSL: –
Currency: BRL (R$)
Currency Position: left
Thousand Separator: .
Decimal Separator: ,
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)


### WC Pages ###

Base da loja: #8 - /loja/
Carrinho: #9 - /carrinho/
Finalizar compra: #10 - /finalizar-compra/
Minha conta: #11 - /minha-conta/

### Theme ###

Name: Wootique
Version: 1.6.14
Author URL: http://www.woothemes.com
Child Theme: ❌ – Caso você esteja personalizando o WooCommerce em um tema pai que você não desenvolveu
então recomendamos utilizar um tema filho. Veja como criar um tema filho

WooCommerce Support: ✔

### Templates ###

Overrides: –
`
```
</details>

**Comment by mikejolley** [comment]
This is not a bug, your theme merely lacks styling of h2 elements. I'm not sure if WooTique is actively maintained right now but pinging @jeffikus  I think it's https://github.com/woocommerce/wootique/issues/32

Fix the overlap with CSS:

`
figure.woocommerce-product-gallery__wrapper {
    margin-left: 0;
}
`

And style the h2 elements in the shop loops with a different font size based on needs.

**Comment by jeffikus** [comment]
Just letting you know that this was resolved in Wootique 1.6.15 :)
